.github/workflows/build-beta.yml

```diff
@@ -0,0 +1,142 @@
+---
+name: build-beta
+
+on:
+  push:
+    branches:
+      - build-\d+$
```

Comment:
0/5 I thought we decided to set this as `build-beta` based on https://github.com/mattermost/mattermost-mobile/pull/7087#discussion_r1151918678

Reply:
I thought this was only for the workflow name and file. 
This functionality did not exist on CircleCI though and we do not want to change the functionality together with the Github Actions migration . 
In order to enable this mattermod also need to be changed since the branches are created from there and we do not want that

Reply:
how is mattermod involved in this?

Reply:
Can you share the exact branches you want the beta build to be triggered ?

Reply:
Nice . Thanks mate . Will tackle those early in the morning so that hopefully we merge tomorrow . 
Cheers 🚀